api/scripts/getOldPolkadotRewardEras.js

```diff
@@ -109,22 +109,27 @@ function parseRewards(
         validators: {}
       }
       const validators = {}
+      const allValidatorPreferencesUnwrapped = Object.fromEntries(
+        Object.entries(allValidatorPreferences).map(([address, value]) => {
+          return [address, value.toJSON()]
+        })
+      )
 
       Object.entries(eraValidators).forEach(([validatorId, exposure]) => {
         const validatorPoints = allValidatorPoints[validatorId]
-          ? BN(allValidatorPoints[validatorId])
+          ? new BN(allValidatorPoints[validatorId])
           : ZERO
-        const validatorCommission = allValidatorPreferences[validatorId]
-          ? BN(allValidatorPreferences[validatorId].commission)
+        const validatorCommission = allValidatorPreferencesUnwrapped[validatorId]
+          ? new BN(allValidatorPreferencesUnwrapped[validatorId].commission)
           : ZERO
-        const available = BN(eraReward)
-          .times(validatorPoints)
-          .dividedBy(eraPoints)
+        const available = new BN(eraReward)
+          .mul(validatorPoints)
+          .div(eraPoints)
```

Comment:
oh, I see. Just a different library for doing the same

Reply:
yep, but for passing the eras back to the api library we need to use that library

Reply:
then why not just use `bignumber`? What am I missing?

Reply:
because I didn't think we should have 2 separate libraries that do the same in this script